### ISSUE TITLE
tetra-triangulation , make , error : 

### ISSUE THREAD

**Issue by goometasoft** [autonomousvision/gaussian-opacity-fields#8]
```
win11  wsl2  ubuntu-22.04 cuda-12.1

(cuda121) root@LZH5:/mnt/e/gaus/submodules/tetra-triangulation# make
[ 33%] Building CXX object CMakeFiles/tetranerf_cpp_extension.dir/src/triangulation.cpp.o
In file included from /mnt/e/AI/A2Q/240426/wsl_cuda121/submodules/tetra-triangulation/src/triangulation.cpp:1:
/mnt/e/AI/A2Q/240426/wsl_cuda121/submodules/tetra-triangulation/src/triangulation.h:1:10: 
fatal error: cuda_runtime.h: No such file or directory
    1 | #include <cuda_runtime.h>
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/tetranerf_cpp_extension.dir/build.make:76: 
CMakeFiles/tetranerf_cpp_extension.dir/src/triangulation.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:100: CMakeFiles/tetranerf_cpp_extension.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

**Comment by Eyoahz1243fei** [comment]
> ```
> win11  wsl2  ubuntu-22.04 cuda-12.1
> 
> (cuda121) root@LZH5:/mnt/e/gaus/submodules/tetra-triangulation# make
> [ 33%] Building CXX object CMakeFiles/tetranerf_cpp_extension.dir/src/triangulation.cpp.o
> In file included from /mnt/e/AI/A2Q/240426/wsl_cuda121/submodules/tetra-triangulation/src/triangulation.cpp:1:
> /mnt/e/AI/A2Q/240426/wsl_cuda121/submodules/tetra-triangulation/src/triangulation.h:1:10: 
> fatal error: cuda_runtime.h: No such file or directory
>     1 | #include <cuda_runtime.h>
>       |          ^~~~~~~~~~~~~~~~
> compilation terminated.
> make[2]: *** [CMakeFiles/tetranerf_cpp_extension.dir/build.make:76: 
> CMakeFiles/tetranerf_cpp_extension.dir/src/triangulation.cpp.o] Error 1
> make[1]: *** [CMakeFiles/Makefile2:100: CMakeFiles/tetranerf_cpp_extension.dir/all] Error 2
> make: *** [Makefile:91: all] Error 2
> ```

you can try this "sudo apt-get install nvidia-cuda-toolkit". But after you make it(bulid tetra-triangulation), you are supposed to change your CUDA version back by "sudo apt-get remove --purge nvidia-cuda-toolkit". Plrease note that this operation is risky.

**Comment by ingra14m** [comment]
Hi @goometasoft, I encountered the same problem. I solved it by `export CPATH=/usr/local/cuda-11.6/targets/x86_64-linux/include:$CPATH`.
